area_mapping.py

```diff
@@ -98,7 +99,7 @@ def overlay_mapping(img_path: str, tree_mask: np.ndarray, water_mask: np.ndarray
     water_and_coast_mask = np.logical_or(water_mask == 1, coast_mask == 1).astype(np.uint8)
 
     blueprints = hf.get_buildings()
-    buildings = hf.place_buildings(blueprints, 
+    buildings, building_mask = hf.place_buildings(blueprints, 
```

Comment:
Ich denke, wir sollten die Mengen in buildings.json angeben (vorallem da es eigentlich die Hub für die ganzen Parameter ist) und ich denke, es wäre besser, wenn wir die Masken in der Funktion zuweisen.

Reply:
In diesem Zusammenhang:
```python
amounts={
             "res-building 1": 2, 
             "res-building 2": 1, 
             "workshop": 1, 
             "HEP-Plant": 2, 
             "lumberjack": 1, 
             "port": 1, 
             "mine": 1}, 
          masks={
              "zero": zero_mask,
              "coast": coast_mask, 
              "inland": inland_mask, 
              "forest_edge": forest_edge_mask, 
              "water_and_coast": water_and_coast_mask}
         )
```

Reply:
Was meinst du mit masks in der Funktion zuweisen?